### DISCUSSION TITLE
rotate seismic property of single crystal

### DISCUSSION THREAD

**Discussion by lwlgeo** [mtex-toolbox/mtex#472] [Ask Anything] [answered]
Hi, guys,
I am trying to plot a single crystal seismic properties. However, it seems like all the seismic property is ploted with a axis (parallel to X) in the east and c axis (parallel to Z) in the outofplane automatically. how can i rotate the seismic property to make the c axis in the east and a axis in the outofplane?

Best
Wenlong

**Comment by ralfHielscher** [comment, marked as answer]
Hi Wenlong,

in order to accomplish this you need to rotate the tensor, e.g. by 
```matlab
rot = rotation.map(xvector,zvector,zvector,xvector)
rotTensor = rot * myTensor;
plot(myTensor)
```
We are working on a more native support of such plotting conventions.

Ralf.

**Comment by lwlgeo** [comment]
Hi Ralf,
Thank you very much, it works!


Best,
Wenlong










At 2020-04-13 13:49:01, "Ralf Hielscher" <notifications@github.com> wrote:

Hi Wenlong,

in order to accomplish this you need to rotate the tensor, e.g. by

rot = rotation.map(xvector,zvector,zvector,xvector)
rotTensor = rot * myTensor;
plot(myTensor)

We are working on a more native support of such plotting conventions.

Ralf.
